syftbox/tui/app.py

```diff
@@ -0,0 +1,55 @@
+from pathlib import Path
+from typing import Any
+
+from textual.app import App
+from textual.widgets import Footer, Header, TabbedContent, TabPane
+
+from syftbox.lib import Client
+from syftbox.tui.widgets.api_widget import APIWidget
+from syftbox.tui.widgets.datasites_widget import DatasiteSelector
+from syftbox.tui.widgets.home_widget import HomeWidget
+from syftbox.tui.widgets.sync_widget import SyncWidget
+
+
+class SyftBoxTUI(App):
+    CSS_PATH = Path(__file__).parent.parent / "assets" / "tui.tcss"
+    BINDINGS = [
+        ("h", "switch_tab('Home')", "Home"),
+        ("a", "switch_tab('APIs')", "APIs"),
+        ("d", "switch_tab('Datasites')", "Datasites"),
+        ("s", "switch_tab('Sync')", "Sync"),
```

Comment:
The default shortcut (ctrl+q) is closing the whole window for me (vscode or iterm with tabs). Can we add another shortcut?
```suggestion
        ("s", "switch_tab('Sync')", "Sync"),
        ("q", "quit", "Quit"),
```

Reply:
ah nice, i had the same issue. adding it